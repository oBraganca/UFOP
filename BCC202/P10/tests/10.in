10
-8 71 -21 -15 26 -22 16 13 35 22 21 -17 23 6 7 -1 -14 -27 9 17 -23 11 -28 30 20 12 -3 -19 15 8 1 -30 0 -11 -9 -31 4 18 -5 -29 -7 5 -35 -34 -25 -4 31 25 34 14 -8 -16 29 10 3 32 -20 -18 -32 19 -6 -33 33 -24 -10 27 28 -2 24 -13 2 -26 -12
-40 43 -19 1 7 -11 19 -20 12 17 15 -7 5 -15 11 8 -3 -9 -16 -13 10 -21 -18 21 -12 2 16 -14 14 -8 20 13 -17 9 4 18 -4 6 -10 -5 0 -2 -6 -1 3
13 23 1 6 -8 -10 -3 -9 -2 8 5 0 10 4 -5 2 -7 7 -11 9 11 -4 -6 3 -1
-42 63 -5 10 -3 -8 -2 8 -22 0 -17 2 -12 22 1 5 -9 25 29 -14 21 -11 -10 -13 4 19 31 3 -28 18 13 -24 -18 27 23 -7 14 -19 -20 17 -23 16 26 -30 -4 20 6 -29 7 -21 28 15 -6 -27 30 -1 -25 -26 11 -31 24 12 -15 9 -16
-28 30 12 9 -11 -13 11 4 -3 -5 7 -8 -9 -1 3 6 -7 -14 -15 13 1 14 -10 10 2 -2 8 -4 0 -6 5 -12
-9 26 -10 2 -11 -12 10 -2 -1 6 -9 11 8 -4 -3 -6 12 -7 5 1 7 9 4 0 -8 -5 -13 3
1 45 12 6 14 -15 -20 -5 -11 16 -18 8 10 7 11 1 -1 -8 -7 19 5 -13 -9 13 -2 -4 -16 20 4 -6 21 -17 -12 22 15 0 -21 17 9 -10 -3 -14 -19 18 3 -22 2
-5 21 -4 5 -3 -1 10 3 -8 8 -2 -5 6 2 -9 0 -6 1 7 9 -7 4 -10
21 56 -2 -28 6 25 10 24 -9 23 -17 -3 3 -4 27 15 -15 -7 8 14 22 20 11 18 26 -8 -24 2 -19 17 -16 -20 -23 -10 9 -22 7 -26 19 -25 -6 -21 12 16 -11 13 -12 5 -27 -13 -1 0 1 21 -5 -14 -18 4
-6 20 2 8 7 5 6 -7 3 -5 -1 -2 -9 -10 -8 -6 -4 0 4 9 -3 1
