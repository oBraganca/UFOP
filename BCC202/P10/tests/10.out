49 6
43 5
23 5
63 6
30 4
9 5
14 5
10 4
52 6
14 2
